src/support/docs.js

```diff
@@ -126,7 +126,7 @@ module.exports = ({
     }
 
     const processJSDoc = async () => {
-        await JSDocEngine.render(jsdoc, jsdocOut)
+        await JSDocEngine.render(jsdoc, jsdocOut, data)
```

Comment:
As @ismay mentioned can we change this to `data.docsite.jsdoc2md`?  I also agree the `data` name is confusing (and I'm the one who made it up, I think...) but that's probably out of scope